annonatate/flaskserver.py

```diff
@@ -35,7 +35,7 @@
                   GITHUB_CLIENT_SECRET = client_secret
                   )
 Session(app)
-github = GitHub(app)
+github = GitHubAnno(app)
```

Comment:
I am very confused by this. It is working but I am not sure why it is.

Reply:
GitHubAnno is a new class which inherits from the GitHub class defined in flask-github. It can do everything that the GitHub  class could do, plus it has newly-defined methods. This allows encapsulating all of annonatate's github communication in a single class.

Reply:
That is what I thought but wasn't sure how it "knew" we were passing the app but I am not that much an expert on Inheritance.